internal/raft/leader.go

```diff
@@ -15,10 +15,11 @@ import (
 // Empty append entries request are also called heartbeats.
 // The data that goes in the append entries request is determined by
 // existance of data in the LogChannel channel.
-
-// TODO: Log errors.
 func startLeader(node *Node) {
 
+	node.log.
+		Debug().
+		Str(node.PersistentState.SelfID.String(), "started leader election proceedings")
```

Comment:
```suggestion
	node.log.Debug().
	        Str("self-id", node.PersistentState.SelfID.String()).
		Msg("start leader election proceedings")
```
Without `Msg` or `Msgf`, nothing will be logged.

Reply:
Ah alright :D